styles/styles.css

```diff
@@ -707,9 +706,37 @@ main .columns-container {
   opacity: 0;
 }
 
-/* mobile code start here */
+main .why-us h2 {
+  padding-bottom: 20px;
+}
+
+main .why-us p{
+ font-size: 18px!important;
```

Comment:
Use ascending specificity in rule selectors to avoid having to set `!important`.